src/main/java/com/ibm/watson/modelmesh/ModelMesh.java

```diff
@@ -3389,6 +3427,9 @@ protected Object invokeModel(final String modelId, final Method method, final Me
         final String threadNameBefore = setThreadName(curThread, "invoke-" + hopType + '-' + modelId);
         ModelRecord mr = null;
         try {
+            //System.err.println("***");
+            //System.err.println("persisting "+Arrays.toString(args));
+            //System.err.println("***");
```

Comment:
if this is useful for debugging later on, maybe use `logger.debug()` or `logger.info()` instead of writing to syserr

Reply:
this was just a leftover from initial debugging session, I've dropped it.